w10h01/test/pgdp/trials/TrailOfTheGrassesTest.java

```diff
@@ -0,0 +1,38 @@
+package pgdp.trials;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+
+public class TrailOfTheGrassesTest {
+    @Test
+    void testExample() {
+        // Example from main() of TrialOfTheGrasses.java
+        TrialOfTheGrasses.TreeNode<Integer> nodes = new TrialOfTheGrasses.TreeNode<>(1,
+                new TrialOfTheGrasses.TreeNode<>(2,
+                        new TrialOfTheGrasses.TreeNode<>(3, new TrialOfTheGrasses.TreeNode<>(4),
+                                new TrialOfTheGrasses.TreeNode<>(5))),
+                new TrialOfTheGrasses.TreeNode<>(5),
+                new TrialOfTheGrasses.TreeNode<>(6, new TrialOfTheGrasses.TreeNode<>(7),
```

Comment:
```suggestion
                new TrialOfTheGrasses.TreeNode<>(6,
                        new TrialOfTheGrasses.TreeNode<>(7),
```

Reply:
Is this the formatting from your IDE?

Reply:
No, this is just for simplicity so every node is in it's line.